ui/client/http/HttpService.ts

```diff
@@ -160,6 +160,13 @@ class HttpService {
       .catch(error => this.addError("Cannot fetch status", error))
   }
 
+  fetchProcessesDeployments(processId: string) {
+    return api.get<{performedAt: string, action: "UNARCHIVE" | "ARCHIVE" | "CANCEL" | "DEPLOY"}[]>(`/processes/${processId}/deployments`)
```

Comment:
Action can be represented as Enum.

Reply:
it's temporary, it will change soon, makes no sense to change it now